scripts/makefiles/proto.mk

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+###                         Proto & Mock Generation                         ###
+###############################################################################
+proto-help:
+	@echo "proto subcommands"
+	@echo ""
+	@echo "Usage:"
+	@echo "  make proto-[command]"
+	@echo ""
+	@echo "Available Commands:"
+	@echo "  all        Run proto-format and proto-gen"
+	@echo "  gen        Generate Protobuf files"
+	@echo "  gen-1      Generate Protobuf files (old relic)"
+	@echo "  doc        Generate proto docs"
+
+proto: proto-help
+proto-all: proto-gen
+
+# todo : @AJ needs to address this after removing third_party. Refer this for removal https://github.com/osmosis-labs/osmosis/blob/188abfcd15544ca07d468c0dc0169876ffde6079/scripts/makefiles/proto.mk#L39
```

Comment:
Is this TODO for now?

Reply:
Not for now. To be addressed by @ajansari95 in his cleanup PR